scylla/src/transport/locator/replication_info.rs

```diff
@@ -22,19 +22,31 @@ impl DatacenterNodes {
             rack_count: 0,
         }
     }
+
+    pub(crate) fn get_dc_ring(&self) -> &TokenRing<Arc<Node>> {
+        &self.dc_ring
+    }
 }
 
 #[derive(Clone)]
 pub struct ReplicationData {
-    pub(crate) global_ring: TokenRing<Arc<Node>>,
+    global_ring: TokenRing<Arc<Node>>,
     unique_nodes_in_global_ring: Vec<Arc<Node>>,
 
-    pub(crate) datacenters: HashMap<String, DatacenterNodes>,
+    datacenters: HashMap<String, DatacenterNodes>,
 }
 
 static EMPTY_DATACENTER_NODES: DatacenterNodes = DatacenterNodes::new_empty();
 
 impl ReplicationData {
+    pub(crate) fn get_datacenters(&self) -> &HashMap<String, DatacenterNodes> {
```

Comment:
why not `pub`
/havaker

Reply:
Because `pub(crate)` implies instantly that the user can not see that function, so I can e.g. modify its signature as desired.